utils/.vscode/settings.json

```diff
@@ -10,6 +10,6 @@
         "**/node_modules": true
     },
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```

Comment:
Not sure why this changed, but I noticed some of our other extensions are like that so I just left it.